CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## v0.10.0 / 2018-06-29
+
+BREAKING CHANGES:
+* Flags now use the Kingpin library, and require double-dashes. #222
+
+This also changes the behavior of boolean flags.
+
+    --collector.global_status
```

Comment:
backticks around these

Reply:
Ah it's already indented as a code block, but it might be good to explicitly explain that one is to enable a collector and the other one to disable it.

Reply:
4 space indent causes a markdown block, or do you prefer 3-backticks format?

Reply:
See second comment above :)